config/development-dist.js

```diff
@@ -8,6 +8,9 @@ module.exports = {
                 database: 'boilerplate',
             },
         },
+        frontend: {
+            history: 'createHashHistory',
```

Comment:
In production, using `createBrowserHistory` will give us urls like:
`http://our_domain.com/products`